rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -174,48 +175,6 @@ def define_graph(self):
     def get_handle(self):
         return self._handle
 
-    def copyImage(self, array):
```

Comment:
I think we need to retain all these copy API since we get performance only with these API

Reply:
We can introduce these in future APIs Fiona. Again this requires changes & testing as API changes needed